content/en/armory-enterprise/installation/aurora-install.md

```diff
@@ -84,6 +85,7 @@ Register your Armory Enterprise environment so that it can communicate with Armo
    - `read:infra:data`
    - `exec:infra:op`
    - `read:artifacts:data`
+   - `connect:agentHub`
```

Comment:
`connect:agentHub` is the only required scope the rest of these are dropped now.